26-register_publication/creator_selects_reference_type_part_of_book_report_and_selects_subtype.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Creator selects Reference type Part of book/report and selects subtype
+
+    @395
+    Scenario: Creator sees fields for subtypes compatible with Reference type Part of book/report
+        Given Creator begins registering a Registration in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference Type "Part of book/report"
+        And they select the Registration Subtype "<Subtype>"
+        Then they see an information box describing that a Container must be published first
+        And they see a disabled field for "DOI to resource"
+        And they see a Search box for "<Container>"
+        And they see a preselected value for Peer review "Not peer reviewed"
+        And they see field for Page number from
+        And they see field for Page number to
+        And they see the Norwegian Science Index (NVI) evaluation status
```

Comment:
Avventer info fra @terjehell/Hege om både kapittel i bok og rapport kan inngå i NVI. Hvis det ikke er tilfelle deler vi heller opp i 2 scenario: ett for `Kapittel i bok`, og ett for `Kapittel i rapport`

Reply:
Oppdatering: bare kapittel i bok kan inngå i NVI, så denne skal deles i to